README.md

```diff
@@ -1,11 +1,15 @@
-# tdl-warmup-java
+# tdl-runner-java
 
 
 ## 1. Requirements
 
 - `Java 1.8`
 
-## 2. How to start
+## 2. Import project
+
+You can import this project either via Maven (`pom.xml`) or Gradle (`build.gradle`) into your IDE (if either of these are supported by your IDE).
```

Comment:
The main IDEs are IntelliJ and Eclipse, we need to tell the participants if they need to install plugins. (I know they do for Eclipse gradle)